### ISSUE TITLE
Docker: Replace python3-pip with get-pip.py for Python3

### ISSUE THREAD

**Issue by Dr-Bean** [SynoCommunity/spksrc#1724]
Docker currently uses python3-pip, which probably is `pip 1.5.6`. Let's see if we can switch to the same approach as used for Python2, which makes updating `pip` a lot easier.
Untested, but presumably working:
```
RUN wget https://bootstrap.pypa.io/ez_setup.py -O - | python3
RUN wget https://bootstrap.pypa.io/get-pip.py -O - | python3
```

**Comment by Dr-Bean** [comment]
...and if pip doesn't install wheel by itself by now, the Dockerfile also needs `RUN pip3 install wheel`

**Comment by Dr-Bean** [comment]
Fixed with 6d78b7909a2de85e0720c06b339f121e55e99f9d
